sentry-contrib-native-sys/build.rs

```diff
@@ -21,22 +21,67 @@ use std::{
     path::{Path, PathBuf},
 };
 
+#[derive(Copy, Clone)]
+enum Backend {
+    Inproc,
+    Crashpad,
+    Breakpad,
+    None,
+}
+
+fn get_target_default(target_os: &str) -> Backend {
+    match target_os {
+        "windows" | "macos" => Backend::Crashpad,
+        "linux" => Backend::Breakpad,
+        "android" => Backend::Inproc,
+        _ => Backend::None,
+    }
+}
+
+/// Gets the backend we want to use, see https://github.com/getsentry/sentry-native#compile-time-options
+/// for details
+fn get_backend(target_os: &str) -> Backend {
+    if cfg!(feature = "backend-none") {
+        return Backend::None;
+    }
+
+    if cfg!(feature = "backend-crashpad")
+        && (target_os == "macos" || target_os == "windows" || target_os == "linux")
+    {
+        return Backend::Crashpad;
+    }
+
+    if cfg!(feature = "backend-breakpad") && (target_os == "windows" || target_os == "linux") {
```

Comment:
Breakpad isn't supported yet on Windows.
Again, I'm gonna fix this in a moment.